transaction-service/src/main/java/pl/adamboguszewski/transaction/service/application/TransactionService.java

```diff
@@ -0,0 +1,33 @@
+package pl.adamboguszewski.transaction.service.application;
+
+import org.springframework.stereotype.Service;
+import pl.adamboguszewski.transaction.service.api.transaction.CreateTransactionRequest;
+import pl.adamboguszewski.transaction.service.api.transaction.CreateTransactionResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// [TODO] Fill the details of this class.
+@Service
+public class TransactionService {
+
+    public List<Transaction> getAll() {
+        return new ArrayList<>();
+    }
+
+    public Transaction getById(Long id) {
+        return null;
```

Comment:
Returning nulls is very very very bad design. Read about Optional and change function type to Optional<Transaction>, return Optional.empty() for now;